lib/src/connection.dart

```diff
@@ -104,6 +107,14 @@ class PostgreSQLConnection extends Object
   /// to [Notification.processID].
   Stream<Notification> get notifications => _notifications.stream;
 
+  /// Stream of server messages
+  ///
+  /// Listen to this [Stream] to receive events for all PostgreSQL server messages
+  ///
+  /// This includes all messages whether from Extended Query Protocol, Simple Query Protocol
+  /// or Streaming Replication Protocol.
+  Stream<ServerMessage> get messages => _messages.stream;
```

Comment:
I'm not sure if this should be on the public API. Just to be sure, please annotate with https://pub.dev/documentation/meta/latest/meta/internal-constant.html

Reply:
Similar to the other comment, when using Streaming Replication Protocol, this is the only way for the user to listen to messages. 

I'm not sure what be a better alternative. 

I could add the internal annotation to both `addMessage` and `messages` but they'd be the only option for utilizing the Replication Protocol. 

Please let me know what you'd prefer.

Reply:
Ack.